lms/static/scripts/frontend_apps/components/LMSFilePicker.tsx

```diff
@@ -346,43 +346,53 @@ export default function LMSFilePicker({
       break;
     case 'reload':
       continueButton = (
-        <LabeledButton
+        <Button
           variant="primary"
           onClick={() => computeFilesToDisplay(true /* reload */)}
           data-testid="reload"
         >
           Reload
-        </LabeledButton>
+        </Button>
       );
       break;
     case 'select':
       continueButton = (
-        <LabeledButton
+        <Button
           variant="primary"
           disabled={continueAction.disabled}
           onClick={() => confirmSelectedItem()}
           data-testid="select"
         >
           {continueAction.label}
-        </LabeledButton>
+        </Button>
       );
       break;
   }
 
   const withFileUI = ['fetching', 'fetched'].includes(dialogState.state);
 
   if (dialogState.state === 'fetching' && initialFetch) {
-    return <FullScreenSpinner />;
+    return <SpinnerOverlay />;
   }
 
   return (
     <Modal
-      contentClass={classnames('LMS-Dialog', {
-        'LMS-Dialog--wide LMS-Dialog--tall': withFileUI,
+      classes={classnames({
+        // Set a fixed height on the modal when displaying a list of files.
```

Comment:
It looks like this is a preferred height that is still responsive? I notice the dialog still adjusts min/max height to fit the viewport within certain bounds.

Reply:
Yep! There are "guardrails" in place: modals will never exceed a certain percentage of `vh` in height.